### ISSUE TITLE
incorrect hltpath-to-dataset info in run-366451

### ISSUE THREAD

**Issue by missirol** [xiezhen/brilws#5]
I'm using `brilcalc-v3.7.3`, and I see the following
```
> brilcalc lumi -r 366451 --hltpath HLT_EphemeralZeroBias_v3 --dataset EphemeralZeroBias19

dataset Dataset_EphemeralZeroBias19 and hltpathHLT_EphemeralZeroBias_v3 are not in the same menu
```

This seems incorrect, because the HLT menu used in that run [1] does include the trigger path `HLT_EphemeralZeroBias_v3` and the Dataset `EphemeralZeroBias19` (technically, the path named `Dataset_EphemeralZeroBias19`).

What am I missing ?

[1] https://cmsoms.cern.ch/cms/triggers/hlt_report?cms_run=366451

PS. I can't find the tag for `v3.7.3` in this GitHub repository. Is that intended ?

**Comment by xiezhen** [comment]
Hello,
  the repository is moved to cms-bril/brilws. But it's fine to use this area for ticket. 
  The problem is caused by a wrong indentation. It should be fixed with release 3.7.4 (unannounced). Could you please try with that ?

  The name of the dataset argument can be with or without explicit Dataset_. This is a feature quested by issue #3.

thanks,
Zhen

**Comment by missirol** [comment]
Thanks, Zhen. I confirm that with `v3.7.4` I do not see the issue, the output of `brilcalc` is as expected.

In the future I will use `cms-bril/brilws` to report issues, if that's okay.
